components/PageNavigation.vue

```diff
@@ -194,11 +194,6 @@
     .nav-link {
       color: $mediumgrey;
       text-decoration: none;
-      transition: $standard-transition;
```

Comment:
Did you remove this line by mistake? This removed the smooth transition of the color, which I liked.